<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataSelect</class>
 <widget class="QDialog" name="dataSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1580</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,0,4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>帧选择</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="placeholderText">
           <string>搜索</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="paramTreeWidget">
          <column>
           <property name="text">
            <string>表号_名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>值</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>条件选择</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>触发条件：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="selectmenufrm" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>380</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>延    时：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="delayBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>999999999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>ms</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>发送时机：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sendType">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>周期发送</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>立即发送</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>结束条件：</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="1">
            <widget class="selectmenufrm" name="widget_2" native="true">
             <property name="minimumSize">
              <size>
               <width>380</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>目标设备</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QTreeWidget" name="targetTreeWidget">
             <column>
              <property name="text">
               <string>分系统/设备名称</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>设备类型</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>设备描述</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>UUID</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>已选帧</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTreeWidget" name="selectedDataTree">
          <column>
           <property name="text">
            <string>流程</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>值</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>参数变化</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>触发条件</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>结束条件</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>延时</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>发送时机</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>目标设备</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>table_num</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>dev_uuid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>start_table_num</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>end_table_num</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>selectmenufrm</class>
   <extends>QWidget</extends>
   <header location="global">selectmenufrm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
